lib/universal/mapper.js

```diff
@@ -14,6 +14,7 @@ var lookup = require('obj-case');
 var parse = require('url').parse;
 var pick = require('@ndhoule/pick');
 var values = require('@ndhoule/values');
+var find = require('obj-case');
```

Comment:
@myclamm this was just `require`'d above as `lookup` -- just use that instead of re-requiring